workflows/phaseimpute/tests/test_all.nf.test.snap

```diff
@@ -11,13 +11,13 @@
                 "imputation/stitch/concat/NA20359.vcf.gz.tbi",
                 "imputation/stitch/concat/versions.yml"
             ],
-            927
+            1786
```

Comment:
Do you know why the imputation give more variants now ?

Reply:
Initially it used only chr22 posfile. Now it uses chr21 and chr22.